src/workerd/server/server.c++

```diff
@@ -1490,32 +1674,38 @@ public:
 
           TimerChannel& timerChannel = service;
 
-          auto loopback = kj::refcounted<Loopback>(*this, kj::str(id));
+          auto loopback = kj::refcounted<Loopback>(*this, kj::str(idPtr));
           Worker::Lock lock(*service.worker, asyncLock);
           // We define this event ID in the internal codebase, but to have WebSocket Hibernation
           // work for local development we need to pass an event type.
           static constexpr uint16_t hibernationEventTypeId = 8;
-          auto newActor = kj::refcounted<Worker::Actor>(
-              *service.worker, kj::none, kj::str(id), true, kj::mv(makeActorCache),
-              className, kj::mv(makeStorage), lock, kj::mv(loopback),
-              timerChannel, kj::refcounted<ActorObserver>(), kj::none, hibernationEventTypeId);
+
+          actorContainer->actor.emplace(
+              kj::refcounted<Worker::Actor>(
+                  *service.worker, actorContainer->getTracker(), kj::str(idPtr), true,
+                  kj::mv(makeActorCache), className, kj::mv(makeStorage), lock, kj::mv(loopback),
+                  timerChannel, kj::refcounted<ActorObserver>(), actorContainer->tryGetManagerRef(),
+                  hibernationEventTypeId));
 
           // If the actor becomes broken, remove it from the map, so a new one will be created
           // next time.
-          onBrokenTasks.add(onActorBroken(newActor->onBroken(), id.asPtr()));
+          auto& actorRef = KJ_REQUIRE_NONNULL(actorContainer->actor);
+          auto& entry = onBrokenTasks.findOrCreateEntry(actorContainer->getKey(), [&](){
+            return kj::HashMap<kj::String, kj::Maybe<kj::Promise<void>>>::Entry {
+              kj::str(actorContainer->getKey()), kj::none
+            };
+          });
+          entry.value =
+              onActorBroken(actorRef->onBroken(), *actorContainer).eagerlyEvaluate(nullptr);
```

Comment:
Using `eagerlyEvaluate(nullptr)` means we silently ignore any exceptions. Previously, with the `TaskSet`, exceptions would be logged by `taskFailed()`. We should similarly pass an error handler to `eagerlyEvaluate` which logs exceptions.